python/solid_dmft/dmft_tools/initial_self_energies.py

```diff
@@ -306,7 +305,7 @@ def _set_loaded_sigma(sum_k, loaded_sigma, loaded_dc_imp, general_params):
                      for channel in loaded_dc_shell}
                     for calc_dc_shell, loaded_dc_shell in zip(sum_k.dc_imp, loaded_dc_imp)]
     freq = 'w' if general_params['solver_type'] in ['ftps'] else 'iw'
```

Comment:
This can probably be removed.

Reply:
fixed in [4ae8e81](https://github.com/TRIQS/solid_dmft/pull/14/commits/4ae8e81bda197b24cf35273606df9b6fff92879e)